resources/session03/wsgi/calculator.py

```diff
@@ -0,0 +1,79 @@
+
+
+def num(s):
+    # return some sort of number from a string
+    try:
+        return int(s)
+    except ValueError:
+        return float(s)
```

Comment:
I actually tried this function with an invalid string.  It returned a ValueError that I tried to catch in the calling function.  Should I be catching it in this local function instead?

Sent from my iPad

On Jan 25, 2016, at 9:31 PM, Cris Ewing <notifications@github.com> wrote:

In resources/session03/wsgi/calculator.py:

> @@ -0,0 +1,79 @@
> +
> +
> +def num(s):
> +    # return some sort of number from a string
> +    try:
> +        return int(s)
> +    except ValueError:
> +        return float(s)
What happens with this if I give it the string a as input?

—
Reply to this email directly or view it on GitHub.